History.md

```diff
@@ -1,6 +1,15 @@
 unreleased
 ==========
 
+  * deps: statuses@2.0.0
+    - Drop support for Node.js 0.6
```

Comment:
These sub changes are not jsut all of the ones in the dependency, but rather which ones will actually apply to the users of _this_ module, or none if there are no changes this dependency update will make to this module's API.

As an example, this module does not support Node.js 0.6, so the dropping of that in statuses (which is a major version bump) is not relevant to the users of Express.